draft-ietf-tls-esni.md

```diff
@@ -998,6 +1004,9 @@ If the client-facing server rejected ECH, or if the first ClientHello did not
 include an "encrypted_client_hello" extension, the client-facing server
 proceeds with the connection as usual. The server does not decrypt the
 second ClientHello's ECHClientHello.payload value, if there is one.
+Moreover, the server MUST include the "encrypted_client_hello" extension in its
+EncryptedExtensions with the "retry_configs" field set to one or more ECHConfig
+structures with up-to-date keys, as described in {{client-facing-server}}.
```

Comment:
This doesn't have anything to do with HRR. Do we need to restate it from the section here? (I wonder if the server half could also do with an editorial pass to make it more chronological.)

Reply:
We should do such a pass, but let's do it separately.

Reply:
(The server still includes configs upon rejection, even in the event of HRR, right?)

Reply:
It does. The point of my suggestion with the bullet points was instead to have the one sentence apply to both cases.

Reply:
Gotcha. In my reading this section it seemed clear to include here, but we can drop it later on if not needed.